.github/workflows/test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           - 5432:5432
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, '3.10', '3.11']
+        python-version: [3.8, 3.9, '3.10', 3.11, 3.12]
```

Comment:
nit: '3.10' instead of 3.10?

Reply:
The quotes are required for 3.10 because without them YAML evaluates it as a float with the value 3.1. The others are fine.

Reply:
oh wow, can we keep all as strings then?

Reply:
f7bc3f5